driver/registry_base.go

```diff
@@ -316,6 +318,11 @@ func (m *RegistryBase) ScopeStrategy() fosite.ScopeStrategy {
 
 func (m *RegistryBase) newKeyStrategy(key string) (s jwk.JWTStrategy) {
 	if err := jwk.EnsureAsymmetricKeypairExists(context.Background(), m.r, new(jwk.RS256Generator), key); err != nil {
+		var netError net.Error
+		if errors.As(err, &netError) {
```

Comment:
Shouldn't this be `**netError`?

Reply:
In this case I don't believe so, as `net.Error` is an interface. It worked when I tested it locally.

Reply:
Right, that makes sense! Maybe we could add a small test to ensure we don't regress on this in future iterations?

Reply:
Sure, I'll see what I can do.

Reply:
Awesome, thank you!

Reply:
@aeneasr i'm having trouble writing tests for this due to the fact that we call `.Fatalf` as part of the error logging, which inevitably leads to `os.Exit(1)`. Do have any advice/examples on how I might write tests for this? Thanks.

Reply:
It's ok, you can call return explicitly here!